### PR TITLE
add features list to README.md

### PR DESCRIPTION
**Pull request by colemickens**
Quick commit to address #28. It was quick, I'm happy to change any verbiage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ An implementation of VHD [VHD specification](https://technet.microsoft.com/en-us
     2. https://github.com/Azure/azure-sdk-for-go
             This package is used to make Microsoft Azure storage API calls.
 
+# Features
+
+    1. Fast uploads - This tool offers faster uploads than the [Azure Xplat CLI](https://github.com/Azure/azure-xplat-cli).
```
Comment (ahmetalpbalkan) [inline README.md]: un-indent maybe? also the space btw bullet points don't seem necessary when you unindent. also maybe move all the way up before Prerequisites?

Xplat CLI --> Cross-Platform CLI

File: README.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ An implementation of VHD [VHD specification](https://technet.microsoft.com/en-us
...
+
+    2. Efficient uploads - This tool will only upload used (non-zero) portions of the disk.
+
```
Comment (ahmetalpbalkan) [inline README.md]: so no parallelism?

File: README.md (hunk 1 above)
Comment (colemickens) [inline README.md]: indent was to match styling of the line above. Will remove extra line. Never am sure of the CLI branding, will fix too.

File: README.md (hunk 2 above)
Comment (colemickens) [inline README.md]: slow brain, will add.

File: README.md (hunk 1 above)
Comment (ahmetalpbalkan) [inline README.md]: meh, unindenting is better for reading on mobile etc. code blocks will get a scrollbar.

Comment (colemickens) [inline README.md]: Yeah... I thought Markdown was smart enough to not code-block a numbered list, but you're definitely right. I'll unindent. I also used "Azure Cross Platform Command Line" as the branding -- that's what they have on their GitHub repo.

**Comment by colemickens** [comment]
Thanks @ahmetalpbalkan; I've addressed your comments.

**Comment by ahmetalpbalkan** [comment]
:shipit:

**Comment by anuchandy** [comment]
Thanks @colemickens @ahmetalpbalkan .. merging

**Pull request merged**